node/src/BaseClient.ts

```diff
@@ -2845,6 +2847,36 @@ export class BaseClient {
         return this.createWritePromise(createObjectRefcount(key));
     }
 
+    /**
+     * Returns the index of the first occurrence of `element` inside the list specified by `key`. If no
+     * match is found, `null` is returned. If the `count` option is specified, then the function returns
+     * an `array` of indices of matching elements within a list.
+     *
+     * See https://valkey.io/commands/lpos/ for more details.
+     *
+     * since - Valkey version 6.0.6.
```

Comment:
The other docs have the since section after the "@returns" or "Command Response" sections, can you update here and in Transaction.ts?